DOCS/man/input.rst

```diff
@@ -927,6 +927,9 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
     <keep-selection>
         Do not change current track selections.
 
+``show-menu``
```

Comment:
better to use `menu-show` for the consistence in the future (if there would be more commands like `menu-refresh` `menu-close`)

Reply:
There won't be `menu-refresh` and `menu-close`. The menu already refreshs itself on `menu-data` property change, I don't think there's a reason to add a `menu-close` command (it's already modal to video window, how do you trigger it from somewhere else?).

Reply:
that is just an example. they can be menu-xxx for any feature.

Reply:
How about `context-menu`? it might make more sense here.

Reply:
Agreed.